CONTRIBUTING.md

```diff
@@ -7,10 +7,9 @@ For more details on how to contribute to the samples or to the rest of the Quant
 
 ## Asking Questions
 
-Have a question? Rather than opening an issue, please ask on [Stack Overflow](https://stackoverflow.com/questions/tagged/q%23) or [Quantum Computing StackExchange](https://quantumcomputing.stackexchange.com/questions/tagged/q%23) using the tag `q#`.
-
-The active community will be eager to assist you.
-Your well-worded question will serve as a resource to others searching for help.
+Have a question? We have already answered it on the [Q# Development Blog](https://devblogs.microsoft.com/qsharp/), if not feel free to ask us in a comment!
```

Comment:
Looks better, but the "we have already answered it" feels a bit odd to me... I would suggest the following:

Have a question? The `q#` tags on [Stack Overflow](https://stackoverflow.com/questions/tagged/q%23) and [Quantum Computing StackExchange](https://quantumcomputing.stackexchange.com/questions/tagged/q%23) are also great places to ask questions about Q#, Quantum Development Kit and quantum computing in general. You can learn more about our work on the [Q# Blog](https://devblogs.microsoft.com/qsharp/) and ask questions in the comments as well.

(This way we stress that quantum computing in general questions are better on SO, while still sounding welcoming, and point to blog without saying that we've foreseen your questions)

(And keep "However you ask it" sentence, I like it)

Reply:
That was supposed to read "we _may_ have already answered it," but apparently I can't type, sigh. That said, your wording is better anyway, so I've applied that. Thanks!